bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+| eso-temperature               | Number:Temperature         | R          | Eso Temperature Measured on PCB          | Measured inside ESO |
+| eso-fault-code                | String                     | R          | Eso FaultCode                            | See section 4.1.3.1 in Ferroamp-External-API-specifikation |
+| eso-battery-energy-produced   | Number:Energy              | R          | Eso Battery Energy Produced              | Total energy produced by ESO, i.e total energy charged |
+| eso-dc-link-voltage           | Number:ElectricPotential   | R          | Eso Dc Link Voltage                      | DC link voltage as measured by ESO |
+| eso-timestamp                 | DateTime                   | R          | Eso Time Stamp When Message was Published| Time stamp when message was published |
+
+
+| esm-unique-identifier                     | String                     | R          | Esm Unique Identifier                    | Unique identifier of battery. If available, this will be the unique id that the battery reports.|
+| esm-soh                                   | Number:Dimensionless       | R          | Esm System State of Health               | State of Health for ESM |
+| esm-soc                                   | Number:Dimensionless       | R          | Esm System State of Charge               | State of Charge for ESM |
+| esm-total-rated-capacity-all-batteries    | Number:Energy              | R          | Esm Rated Capacity                       | Rated capacity of battery |
+| esm-rated-power-battery                   | Number:Power               | R          | Esm Rated power of battery               | Rated power of battery |
+| esm-status                                | String                     | R          | Esm Status                               | Dependent on battery manufacturer |
+| esm-timestamp                             | DateTime                   | R          | Esm Time Stamp When Message was Published| Time stamp when message was published |
+
+The following channels are available for `Ferroamp` EnergyHub configuration. Please, see Ferroamp documentation for more details.
+
+| Channel Type ID      | Item Type                  | Read/Write | Description                              |
+|---------------------------------------------------|------------|------------------------------------------|
+| request-charge       | String                     | W          | Set charge power, value in Watt          |
+| request-discharge    | String                     | W          | Set discharge power, value in Watt       |
+| request-auto         | String                     | W          | Set auto power. Returning control of batteries to system, value as auto.|
+
+# Full Example
+
+## Thing
+
+```java
+Thing ferroamp:energyhub:myenergyhub [ hostName="energyhub-ip", userName="myUserName", password="myPassword", hasBattery=false ]
```

Comment:
Can you extend the example with other Things

Reply:
20241024
Please, see README.md